doc/source/rllib-algorithms.rst

```diff
@@ -73,6 +80,13 @@ SpaceInvaders  843                              ~300
 
    IMPALA solves Atari several times faster than A2C / A3C, with similar sample efficiency. Here IMPALA scales from 16 to 128 workers to solve PongNoFrameskip-v4 in ~8 minutes.
 
+**IMPALA-specific configs**:
```

Comment:
Maybe mention the default configs/`with_common_config` on this page
<img width="776" alt="screenshot 2018-10-16 11 37 10" src="https://user-images.githubusercontent.com/4529381/47039098-f80fed80-d137-11e8-8342-67c7988b5a39.png">

Reference to `with_common_config` is not on this page but on the `training` which can be pretty confusing...